src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -290,20 +291,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Workspace.VSIntegration">
-      <Version>15.0.198-pre</Version>
-    </PackageReference>
-    <PackageReference Include="VSLangProj110">
-      <Version>11.0.61030</Version>
-    </PackageReference>
-    <PackageReference Include="VSLangProj2">
-      <Version>7.0.5000</Version>
-    </PackageReference>
-    <PackageReference Include="VSLangProj157">
-      <Version>15.7.0</Version>
-    </PackageReference>
-  </ItemGroup>
-  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Workspace.VSIntegration" Version="15.0.198-pre" /> <!-- can this be removed?, upgraded version-->
```

Comment:
how can we know if these should/can be deleted?

Reply:
It will have to be a part of follow up Dev16 work.
I will create an issue and track these.

Reply:
https://github.com/NuGet/Home/issues/7143